src/backend/web/app.js

```diff
@@ -68,4 +68,25 @@ app.use(logger);
 // Include our router with all endpoints
 app.use('/', router);
 
+/**
+ * Error Handler, Pass to front-end
+ */
+// eslint-disable-line no-unused-vars
+app.use((err, req, res, next) => {
+  logger.logger.error(err);
```

Comment:
`logger.logger.error( { error: err } );`

Reply:
For the eslint thing, I think we need `after-used` https://eslint.org/docs/rules/no-unused-vars#args